src/common/mockFactory.ts

```diff
@@ -277,6 +277,7 @@ export default class MockFactory {
             id: `project-${name}`,
             name: `Project ${name}`,
             version: appInfo.version,
+            useSecurityToken: true,
```

Comment:
is there no mock for not using security token? if not, how is that tested?

Reply:
No, but i have a test that modifies the value after getting it from the mock.